src/main/kotlin/no/nav/tilleggsstonader/sak/hendelser/personhendelse/dødsfall/OpprettDødsfallOppgaveTask.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.tilleggsstonader.sak.hendelser.personhendelse.dødsfall
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.tilleggsstonader.kontrakter.felles.ObjectMapperProvider.objectMapper
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.kontrakter.oppgave.Oppgavetype
+import no.nav.tilleggsstonader.sak.opplysninger.oppgave.OppgaveService
+import no.nav.tilleggsstonader.sak.opplysninger.oppgave.OpprettOppgave
+import no.nav.tilleggsstonader.sak.opplysninger.pdl.PersonService
+import no.nav.tilleggsstonader.sak.opplysninger.pdl.dto.FolkeregisteridentifikatorStatus
+import no.nav.tilleggsstonader.sak.opplysninger.pdl.dto.PdlSøker
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = OpprettDødsfallOppgaveTask.TYPE,
+    beskrivelse = "Oppretter oppgave for dødsfall for gitt stønadstype",
+)
+class OpprettDødsfallOppgaveTask(
+    private val personService: PersonService,
+    private val oppgaveService: OppgaveService,
+) : AsyncTaskStep {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    override fun doTask(task: Task) {
+        val data = objectMapper.readValue<DødsfallOppgaveTaskData>(task.payload)
+        val dødsfallHendelse = data.dødsfallHendelse
+
+        val person = personService.hentPersonUtenBarn(dødsfallHendelse.personidenter.first())
+
+        if (personErFortsattDød(person.søker)) {
```

Comment:
Her kan det ev. kommit en korrigering på dødsdato. Burde man hente dødsdato fra PDL i stedet for hendelsen? 

Burde vi sjekke for at personen fortsatt har aktivt vedtak? Eks hvis en saksbehandler plukket opp at personen er død et annet sted før vi opprettet oppgaven. 
Jeg tenker nødvendigvis ikke det, hvis vi oppretter en ekstra oppgave om dette er det ikke noe krise - kommer ikke skje ofte

Reply:
Ja jeg tenker og at det ikke er krise med en ekstra oppgave. Sannsynligheten for at det skal skje er vel fort ganske liten, så den ekstra kompleksiteten er kanskje ikke verdt å ta høyde for 🤷